django/db/backends/utils.py

```diff
@@ -163,6 +232,57 @@ def debug_sql(
             )
 
 
+class AsyncCursorDebugWrapper(AsyncCursorWrapper):
+    # XXX callproc isn't instrumented at this time.
+
+    async def execute(self, sql, params=None):
+        with self.debug_sql(sql, params, use_last_executed_query=True):
+            return await super().execute(sql, params)
```

Comment:
I have my own branch experimenting with async cursors, I _highly recommend_ naming this `aexecute` and `aexecutemany`. You will save yourself al ot of headaches trying to run down why something is weird because some code is trying to call a (sync) execute.